pymc4/distributions/discrete.py

```diff
@@ -58,12 +52,17 @@ class Bernoulli(RandomVariable):
 
     - p: probs
     """
+    def __init__(self, name, p, **kwargs):
+        super().__init__(name, p=p, **kwargs)
+
+    def lower_limit(self):
+        return 0
 
-    def _base_dist(self, p: TensorLike, *args, **kwargs):
-        return tfd.Bernoulli(probs=p, *args, **kwargs)
```

Comment:
Where do you link this to the tf distribution?

Reply:
I do not link yet, figuring out how to deal with transforms first